framework/BaseYii.php

```diff
@@ -533,4 +533,42 @@ public static function getObjectVars($object)
     {
         return get_object_vars($object);
     }
+
+    private static $shutdownHandlers;
```

Comment:
No it could not.
Check `registerShutdownFunction()` it registers `shutdown()` as PHP shutdown function only on first demand, which `null` value indicates.